src/cc/frontends/clang/CMakeLists.txt

```diff
@@ -10,3 +10,4 @@ if(DEFINED BCC_BACKUP_COMPILE)
 endif()
 
 add_library(clang_frontend STATIC loader.cc b_frontend_action.cc tp_frontend_action.cc kbuild_helper.cc ../../common.cc)
+target_link_libraries(clang_frontend bcc-static)
```

Comment:
Drop this commit. Its already merged upstream..

Reply:
It was removed in [commit 29015d7ef003f35f2a4671d67404f9e5647d6b13](https://github.com/iovisor/bcc/commit/29015d7ef003f35f2a4671d67404f9e5647d6b13)

Reply:
Okay so I just looked into this issue. Commit 29015d7  removed the line linking `clang_frontend` to `bcc_static` because it found it unnecessary. Thankfully, it doesn't seem like it had anything to do with the [bug](https://github.com/iovisor/bcc/issues/1621) the commit was trying to resolve, and was just removed cause the author happened to pass by it and figured why not do some cleaning. This means that it is most likely safe to add back in the line that has been removed.